tests/interval/test_interval_length.py

```diff
@@ -0,0 +1,33 @@
+from panda3d.core import *
+from direct.interval.IntervalGlobal import *
+import pytest
+
+
+def test_invalid_sound_interval():
+    load_prc_file_data('', 'audio-library-name p3fmod_audio')
```

Comment:
Why does this test require FMOD?

What happens if FMOD is not available?

What happens to tests running after this one?  If you do this it should at leat be in a try..finally where you unload the page in the `finally`, see `test_track_gui_items` in `tests/gui/test_DirectGuiBase.py` for an example